### PR TITLE
Add How to Think Like a Computer Scientist Learning with Python 3 (Ao…

### PR DESCRIPTION
**Pull request by V205Arduino**
…PS Edition)

Add How to Think Like a Computer Scientist Learning with Python 3 (AoPS Edition) Thank you very much, I hope I did this correctly, if not, please guide me on what I did wrong.

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo
Add resource 
## For resources

https://artofproblemsolving.com/assets/pythonbook/

### Description

A book on learning python3, and thinking like a computer scientist.

### Why is this valuable (or not)?
It is a helpful book on python 3.
### How do we know it's really free?
It is licensed under GNU Free Documentation License
Version 1.3, 3 November 2008

### For book lists, is it a book? For course lists, is it a course? etc.
It is a book that is usable when printed out, and has a table of contents.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by V205Arduino** [comment]
Oh oops...
I will try to fix that now.

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -2000,7 +2000,9 @@ Books on general-purpose programming that don't focus on a specific language are
 * [How to Think Like a Computer Scientist: Learning with Python, Interactive Edition](https://runestone.academy/runestone/books/published/thinkcspy/index.html) - Brad Miller, David Ranum, Jeffrey Elkner, Peter Wentworth, Allen B. Downey, Chris Meyers, Dario Mitchell (3.2)
     * [How to Think Like a Computer Scientist: Learning with Python 1st Edition](https://greenteapress.com/wp/learning-with-python/) - Allen B. Downey, Jeff Elkner, Chris Meyers (2.4) (HTML, PDF)
     * [How to Think Like a Computer Scientist: Learning with Python 2nd Edition](https://openbookproject.net/thinkcs/python/english2e/) - Jeffrey Elkner, Allen B. Downey, Chris Meyers (Using Python 2.x)
+     * [How to Think Like a Computer Scientist: Learning with Python 3 (AoPS Edition)](https://artofproblemsolving.com/assets/pythonbook/) - AoPS Incorporated, Peter Wentworth, Jeffrey Elkner, Allen B. Downey, and Chris Meyers (HTML)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:2003]:
```suggestion
    * [How to Think Like a Computer Scientist: Learning with Python 3 (AoPS Edition)](https://artofproblemsolving.com/assets/pythonbook/) - AoPS Incorporated, Peter Wentworth, Jeffrey Elkner, Allen B. Downey, Chris Meyers (HTML)
```

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -2000,7 +2000,9 @@ Books on general-purpose programming that don't focus on a specific language are
...
     * [How to Think Like a Computer Scientist: Learning with Python 3 (RLE)](https://openbookproject.net/thinkcs/python/english3e/) - Peter Wentworth, Jeffrey Elkner, Allen B. Downey, Chris Meyers [(PDF)](https://www.ict.ru.ac.za/Resources/cspw/thinkcspy3/thinkcspy3.pdf)
+
```
Comment (eshellman) [inline books/free-programming-books-langs.md:2005]: I don't think we need an empty line here

**Review by eshellman — CHANGES_REQUESTED** [review]
minor tweaks

File: books/free-programming-books-langs.md (hunk 2 above)
Comment (V205Arduino) [reply books/free-programming-books-langs.md]: I think I fixed it, thanks!

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Comment by V205Arduino** [comment]
Thank you very much, this repo is very welcoming.

**Pull request merged**